jetty-client/src/main/java/org/eclipse/jetty/client/ConnectionPool.java

```diff
@@ -109,13 +109,22 @@
     interface Multiplexable
     {
         /**
-         * @return the max number of requests multiplexable on a single connection
+         * @return the default max number of requests multiplexable on a connection
          */
         int getMaxMultiplex();
 
         /**
-         * @param maxMultiplex the max number of requests multiplexable on a single connection
+         * @param maxMultiplex the default max number of requests multiplexable on a connection
          */
         void setMaxMultiplex(int maxMultiplex);
+
+        /**
+         * @param connection the multiplexed connection
+         * @param maxMultiplex the max number of requests multiplexable on the given connection
+         */
+        default void setMaxMultiplex(Connection connection, int maxMultiplex)
+        {
+            setMaxMultiplex(maxMultiplex);
```

Comment:
This implementation is wrong.  Setting the max multiplex for a connection is not the same as setting the default for the pool